ui/src/app/index.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import {App} from './app';
+import './i18n';
```

Comment:
```suggestion

import './i18n';
```

separate out the side-effect